run-dev.sh

```
@@ -1,97 +1,97 @@
+echo "✓ All required environment variables are set"
+echo ""
+
+# Check if virtual environment exists
+if [ ! -d ".venv" ]; then
+    echo "📦 Creating Python virtual environment..."
+    python -m venv .venv
+fi
+
+# Activate virtual environment
+echo "🐍 Activating Python virtual environment..."
+source .venv/bin/activate
```

Shellcheck (0.11.0)

[info] Not following: .venv/bin/activate was not specified as input (see shellcheck -x).

(SC1091)